README.md

```diff
@@ -39,246 +34,16 @@ To create the hub from the portal make sure you select the right subscription, r
-  - Resolve the Model contents from the ModelId
-    - If it's a known model, show an ad-hoc UI
-  - For Models without an ad-hoc UI
-    - Show a dynamically generated UI
-  - Devices without ModelId
-    - Show the properties available in the Twin
-
-## Scenario Steps
-
-### 1. Review the existing samples and docs
-
-If you need to learn basic PnP concepts take a minute to familiarize yourself by reading the staging docs available in [review.docs.microsoft.com/azure/iot-pnp](https://review.docs.microsoft.com/azure/iot-pnp?branch=pr-en-us-114965)
-
-To learn how to create a PnP device we have tutorials for C, Node and Python:
-
-- Follow the tutorial [Quickstart: Connect a sample IoT Plug and Play Preview device application running on Linux or Windows to IoT Hub (C)](https://review.docs.microsoft.com/azure/iot-pnp/quickstart-connect-device-c?branch=pr-en-us-114965) to build and run the existing [C device sample](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp/digitaltwin_client/samples)
-
-- Follow the tutorial [Quickstart: Connect a sample IoT Plug and Play Preview device application to IoT Hub (Node.js)](https://review.docs.microsoft.com/azure/iot-pnp/quickstart-connect-device-node?branch=pr-en-us-114965) to build and run the existing [node device sample](https://github.com/Azure/azure-iot-sdk-node/tree/public-preview-pnp/digitaltwins/samples/device/javascript)
-
-- Follow the tutorial [Quickstart: Connect a sample IoT Plug and Play Preview device application to IoT Hub (Python)](https://review.docs.microsoft.com/azure/iot-pnp/quickstart-connect-device-python?branch=pr-en-us-114965) to build and run the existing [Python device sample](https://github.com/Azure/azure-iot-sdk-python/tree/digitaltwins-preview/azure-iot-device/samples/pnp)
-
-- Follow the howto article [HOW-to guides: Manage Models in the repository](https://review.docs.microsoft.com/azure/iot-pnp/howto-manage-models?branch=pr-en-us-114965) to create, publish, share models and manage access.
-
-- Follow the howto article [How-to: Interact with an IoT Plug and Play Preview device that's connected to your solution]( https://review.docs.microsoft.com/azure/iot-pnp/howto-connect-pnp-device-solution?branch=pr-en-us-114965) to create a digital twin route and recieve the digital twin change notifications.
-
-#### Interact with the devices using IoT Explorer
-
-1. Create a new hub instance following the instructions to use the canary environment
-2. Install the private version of IoT Explorer
-3. Create a device registration in IoT explorer and get the device connection string.
-4. Use the connection string to connect the sample devices to hub
-5. IoT Explorer should show the `IoT Plug and Play components` tab
-6. Configure IoT Explorer to load the models from a local folder
-7. Inspect Telemetry, Update properties and invoke commands from IoT Explorer
-
->Note: These steps are valid to validate any PnP device
-
-### 2. Create the Thermostat Device simulator
```

Comment:
Can we include some requirements to create the device simulator? Can we provide sample models?